contracts/modules/Experimental/TransferManager/SignedTransferManagerFactory.sol

```diff
@@ -10,13 +10,18 @@ contract SignedTransferManagerFactory is ModuleFactory {
 
     /**
      * @notice Constructor
+     * @param _setupCost Setup cost of module
+     * @param _usageCost Usage cost of the module
+     * @param _polymathRegistry Address of the Polymath registry
+     * @param _isCostInPoly true = cost in Poly, false = USD
      */
     constructor(
         uint256 _setupCost,
+        uint256 _usageCost,
         address _polymathRegistry,
         bool _isCostInPoly
     )
-        public ModuleFactory(_setupCost, _polymathRegistry, _isCostInPoly)
+        public ModuleFactory(_setupCost, _usageCost, _polymathRegistry, _isCostInPoly)
```

Comment:
General comment is that with this design we can only have one usage fee per module - so any actions in that module must cost the same amount of POLY. If this is OK w/ product that's fine, but not 100% sure if that's the case or not.

Reply:
I do have a similar doubt in my mind, but if you see all our modules they are pretty focussed on one major activity and revolves around that. Still, it is a good question to ask to the business team.